package.json

```diff
@@ -32,6 +32,8 @@
     ]
   },
   "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.0.1",
     "react-redux": "^7.2.0",
```

Comment:
Sorry man 🙏 we missed this in milestone 1.
You should also add `prop-types` because you need to do some prop-types checking, check out the below link.
https://reactjs.org/docs/typechecking-with-proptypes.html